receiver/mongodbatlasreceiver/config.go

```diff
@@ -39,6 +39,7 @@ type Config struct {
 	Granularity                             string                       `mapstructure:"granularity"`
 	Metrics                                 metadata.MetricsSettings     `mapstructure:"metrics"`
 	Alerts                                  AlertConfig                  `mapstructure:"alerts"`
+	Events                                  *EventsConfig                `mapstructure:"events,omitempty"`
```

Comment:
Out of curiosity, why `omitempty` for this one when others do not have the tag?

Reply:
Yeah I was thinking if there was some marshal/unmarshal chain of the config that this would be an issue. But I don't think it necessary after playing around a tad bit more in a couple test cases. Will remove the tag 👍